pandas/core/arrays/string_arrow.py

```diff
@@ -330,17 +330,47 @@ def _str_contains(
             result[isna(result)] = bool(na)
         return result
 
-    def _str_startswith(self, pat: str, na=None):
-        result = pc.starts_with(self._pa_array, pattern=pat)
+    def _str_startswith(self, pat: str | tuple[str, ...], na: Scalar | None = None):
+        if isinstance(pat, str):
+            result = pc.starts_with(self._pa_array, pattern=pat)
+        elif isinstance(pat, tuple) and all(isinstance(x, str) for x in pat):
+            if len(pat) == 0:
+                # mimic existing behaviour of string extension array
+                # and python string method
+                result = pa.array(
+                    np.full(len(self._pa_array), False), mask=isna(self._pa_array)
```

Comment:
Can you use `np.zeros(len(...), dtype=bool)` instead; it has the same asymptotic behavior, but much less overhead.

Reply:
Ok. Fixed in [e55a1e4](https://github.com/pandas-dev/pandas/pull/54943/commits/e55a1e44529496c3c85f0a63d4b117e68864296d)